app/Listeners/CheckoutableListener.php

```diff
@@ -147,7 +147,7 @@ public function onCheckedIn($event)
         $ccEmails = array_filter($adminCcEmailsArray);
         $mailable =  $this->getCheckinMailType($event);
         $notifiable = $this->getNotifiables($event);
-        if  (!$event->checkedOutTo->locale){
+        if  ($event->checkedOutTo->locale){
```

Comment:
Unfortunately, this ignores the `Settings::getSettings()->locale`, which should kick in if the user has no locale set.